dockerfile-image-update/src/test/java/com/salesforce/dockerfileimageupdate/utils/DockerfileGitHubUtilTest.java

```diff
@@ -330,6 +330,7 @@ public void getSearchResultsExcludingOrgWithMostHits() throws Exception {
         Map<String, Boolean> orgsToIncludeOrExclude = new HashMap<>();
 
         assertEquals((dockerfileGitHubUtil.getSearchResultsExcludingOrgWithMostHits("image", contentsWithImage, orgsToIncludeOrExclude, 1000)).get().size(), 2);
+        assertEquals(orgsToIncludeOrExclude.size(), 0);
```

Comment:
Minor: We might want to add a small comment as to why we are checking for this condition.